src/sys/reboot.rs

```diff
@@ -1,48 +1,163 @@
+                RB_POWERDOWN;
+                /// An option allowing the specification of an init program (see
+                /// `init(8)`) other than `/sbin/init` to be run when the system
+                /// reboots.
+                ///
+                /// This switch is not currently available.
+                #[cfg(target_os = "netbsd")]
+                RB_INITNAME;
+                /// By default, the system will halt if `reboot()` is called during
+                /// startup (before the system has finished autoconfiguration), even
+                /// if `RB_HALT` is not specified. This is because `panic(9)`s
+                /// during startup will probably just repeat on the next boot.
+                /// Use of this option implies that the user has requested the
+                /// action specified (for example, using the `ddb(4)` boot reboot
+                /// command), so the system will reboot if a halt is not explicitly
+                /// requested.
+                #[cfg(target_os = "openbsd")]
+                RB_USERREQ;
+                /// Load the symbol table and enable a built-in debugger in the
+                /// system. This option will have no useful function if the kernel
+                /// is not configured for debugging. Several other options have
+                /// different meaning if combined with this option, although their
+                /// use may not be possible via the `reboot()` call. See `ddb(4)` for
+                /// more information.
+                RB_KDB;
+                /// Normally, the disks are sync'd (see `sync(8)`) before the
+                /// processor is halted or rebooted. This option may be useful
+                /// if file system changes have been made manually or if the
+                /// processor is on fire.
+                RB_NOSYNC;
+                /// Normally, the reboot procedure involves an automatic disk
+                /// consistency check and then multi-user operations. `RB_SINGLE`
+                /// prevents this, booting the system with a single-user shell on
+                /// the console. `RB_SINGLE` is actually interpreted by the `init(8)`
+                /// program in the newly booted system.
+                ///
+                /// When no options are given (i.e., `RB_AUTOBOOT` is used), the
+                /// system is rebooted from file /bsd in the root file system of
+                /// unit 0 of a disk chosen in a processor specific way. An automatic
+                /// consistency check of the disks is normally performed (see `fsck(8)`).
+                RB_SINGLE;
+                /// The `bootstr` argument is a string passed to the firmware
+                /// on the machine, if possible, if this option is set.
+                ///
+                /// Currently this is only implemented on the sparc and the sun3 ports.
+                #[cfg(target_os = "netbsd")]
```

Comment:
Let's define it for sparc64 only.  It looks like NetBSD doesn't have a Rust package for sun3, so we don't have to worry about that. https://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/lang/rust/index.html
```suggestion
                #[cfg(all(target_os = "netbsd", target_arch = "sparc64"))]
```

Reply:
This flag and that `bootstr` argument are not removed